utility/repoutil.py

```diff
@@ -139,6 +97,7 @@ def _clone(self, clone_options: CloneOptions):
                 config.set_value('user', 'name', config_username)
 
 
+# This probably isn't worth it's own class anymore
 class ModifiableRepoUtil(RepoUtil):
```

Comment:
I believe that this class and the RepoUtil class should be removed if they aren't used anymore

Reply:
Yeah your right, I forgot this comment there.